src/mbgl/tile/geometry_tile.hpp

```diff
@@ -87,6 +88,9 @@ class GeometryTile : public Tile, public GlyphRequestor, IconRequestor {
     }
 
 private:
+    void markObsolete();
```

Comment:
This `markObsolete` looks like it snuck in from `master`, it's not used on this branch.